README.md

```diff
@@ -1,10 +1,8 @@
-In order to run this app:
- 
-- Install [node.js](https://nodejs.org/en/).
-- Clone the repository.
-- Install dependencies using `npm install`.
-- Make a new app at Heroku, and add the Redis Cloud add-on (free plan) and note the app URL.
```

Comment:
Good call. I was in two minds when I removed it, I'll put it back when I get to a machine.